packages/app/src/components/Common/ImageCropModal.tsx

```diff
@@ -49,6 +49,13 @@ const ImageCropModal: FC<Props> = (props: Props) => {
   const { t } = useTranslation();
   const reset = useCallback(() => {
     if (imageRef) {
+      // Some SVG files may not have width and height properties, causing the render size to be 0x0
+      // Force imageRef to have width and height by create temporary image element then set the imageRef width with tempImage width
+      const tempImage = new Image();
+      tempImage.src = imageRef.src;
+      imageRef.width = tempImage.width;
```

Comment:
is assigning temporary values into imageRef.width and imageRef.height not enough if you just want properties?
something like: 
```
if (imageRef.width == null) { imageRef.width = 0 } ;
if (imageRef.height == null) { imageRef.height = 0 };
```